guides/common/modules/proc_associating-partition-tables-with-disk-encryption.adoc

```diff
@@ -0,0 +1,29 @@
+[id="associating-partition-tables-with-disk-encryption_{context}"]
+= Associating partition tables with disk encryption
+
+{Project} contains partition tables that encrypt the disk of your host using Linux Unified Key Setup (LUKS) during host provisioning.
+Optionally, you can also bind the disk to a Tang server through Clevis for decryption during boot.
+You must associate the partition table with your operating system entry.
+Then, you assign the partition table to your host group or select it manually during provisioning.
+
+Encrypted disks on hosts protect data at rest.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *Provisioning Setup* > *Operating Systems*.
+. Select your {client-os} entry.
+. On the *Partition Table* tab, associate the `{client-provisioning-template-type} default encrypted` to your operating system entry.
+For more information, see xref:creating-partition-tables_{context}[].
+. Create a host group that uses the `{client-provisioning-template-type} default encrypted` partition table.
+For more information, see {ManagingHostsDocURL}Creating_a_Host_Group_managing-hosts[Creating a host group] in _{ManagingHostsDocTitle}_.
+. Decrypt the disk of your host during boot time by using one of the following options:
++
+* LUKS encryption: Add the host parameter `disk_enc_passphrase` as type *string* and your cleartext passphrase of the LUKS container as the value.
```

Comment:
Or actually, how about:
```suggestion
+
For LUKS encryption:
+
** Add the host `disk_enc_passphrase` parameter as type *string*.
** Add your cleartext passphrase of the LUKS container as the value.
```
Or something along these lines. (Written on one line, the instructions feel like a bit too much.)

Reply:
Also a good suggestion. I wonder whether we could simplify this further. I'm not sure we need to mention that it is a host parameter because of the context. I think the user can figure out that the examples are Tang server IP addresses. At least I assume these are IPs. If so, we can use more realistic examples like 192.0.2.0 to 192.0.2.255 (block used for d/s OpenShift docs).

* LUKS encryption: Add the parameter `disk_enc_passphrase` (type **string**): <LUKS-cleartext-passphrase>
* Tang server with Clevis: Add the parameter `disk_enc_tang_servers` (type **array**): `["192.0.2.0"]` or `["{server-example-com}", "192.0.2.0"]`)